src/lib.rs

```diff
@@ -187,6 +193,62 @@ impl TestnetDeploy {
 
         Ok(())
     }
+
+    pub async fn deploy(
+        &self,
+        name: &str,
+        node_count: u16,
+        repo_owner: Option<String>,
+        branch: Option<String>,
```

Comment:
I assume `branch` will be implemented/used in the follow up PR.

Reply:
Yeah, branch and repo owner are used by Ansible to build the custom branch. They aren't used by Terraform, other than to determine whether to create a build VM, which has a much higher specification than node VMs.